package/network/services/dnsmasq/files/dnsmasq.init

```diff
@@ -973,6 +973,16 @@ dnsmasq_start()
 	append_bool "$cfg" rapidcommit "--dhcp-rapid-commit"
 	append_bool "$cfg" scriptarp "--script-arp"
 
+	config_get add_mac "$cfg" add_mac
+	if [ "$add_mac" = "enabled" ] || [ "$add_mac" = "1" ]; then
```

Comment:
I think you should drop allowing `enabled` as a parameter as that could cause confusion and lead the user to think that other parameters also take `enabled`. For example, `ignore_hosts_dir` does not take `enabled` as a parameter: https://github.com/openwrt/openwrt/blob/28ff4a9ebc9671cceb4544efc8d362dd3fef91c5/package/network/services/dnsmasq/files/dnsmasq.init#L1019-L1025

Reply:
Ok, so use "1" as default value for --add-mac without any value and pass all other values?

Reply:
Done!